src/demo/app/demo.component.ts

```diff
@@ -1,10 +1,14 @@
 import { Component } from '@angular/core';
+import { KeyboardShortcutsService } from '../../../dist/keyboard-shortcut.service';
```

Comment:
changes to the demo files here in this branch seems out of place.  Can we do that as a different branch?